catalog/ec2-policies.yaml

```diff
@@ -74,273 +116,65 @@
-      - is4gen.large
-      - is4gen.medium
-      - is4gen.xlarge
-      - m5dn.12xlarge
-      - m5dn.16xlarge
-      - m5dn.24xlarge
-      - m5dn.2xlarge
-      - m5dn.4xlarge
-      - m5dn.8xlarge
-      - m5dn.large
-      - m5dn.metal
-      - m5dn.xlarge
-      - m5n.12xlarge
-      - m5n.16xlarge
-      - m5n.24xlarge
-      - m5n.2xlarge
-      - m5n.4xlarge
-      - m5n.8xlarge
-      - m5n.large
-      - m5n.metal
-      - m5n.xlarge
-      - m5zn.12xlarge
-      - m5zn.2xlarge
-      - m5zn.3xlarge
-      - m5zn.6xlarge
-      - m5zn.large
-      - m5zn.metal
-      - m5zn.xlarge
-      - m6a.12xlarge
-      - m6a.16xlarge
-      - m6a.24xlarge
-      - m6a.2xlarge
-      - m6a.32xlarge
-      - m6a.48xlarge
-      - m6a.4xlarge
-      - m6a.8xlarge
-      - m6a.large
-      - m6a.metal
-      - m6a.xlarge
-      - m6i.12xlarge
-      - m6i.16xlarge
-      - m6i.24xlarge
-      - m6i.2xlarge
-      - m6i.32xlarge
-      - m6i.4xlarge
-      - m6i.8xlarge
-      - m6i.large
-      - m6i.metal
-      - m6i.xlarge
-      - m6id.12xlarge
-      - m6id.16xlarge
-      - m6id.24xlarge
-      - m6id.2xlarge
-      - m6id.32xlarge
-      - m6id.4xlarge
-      - m6id.8xlarge
-      - m6id.large
-      - m6id.metal
-      - m6id.xlarge
-      - p3dn.24xlarge
-      - p4d.24xlarge
-      - r5dn.12xlarge
-      - r5dn.16xlarge
-      - r5dn.24xlarge
-      - r5dn.2xlarge
-      - r5dn.4xlarge
-      - r5dn.8xlarge
-      - r5dn.large
-      - r5dn.metal
-      - r5dn.xlarge
-      - r5n.12xlarge
-      - r5n.16xlarge
-      - r5n.24xlarge
-      - r5n.2xlarge
-      - r5n.4xlarge
-      - r5n.8xlarge
-      - r5n.large
-      - r5n.metal
-      - r5n.xlarge
-      - r6a.12xlarge
-      - r6a.16xlarge
-      - r6a.24xlarge
-      - r6a.2xlarge
-      - r6a.32xlarge
-      - r6a.48xlarge
-      - r6a.4xlarge
-      - r6a.8xlarge
-      - r6a.large
-      - r6a.metal
-      - r6a.xlarge
-      - r6i.12xlarge
-      - r6i.16xlarge
-      - r6i.24xlarge
-      - r6i.2xlarge
-      - r6i.32xlarge
-      - r6i.4xlarge
-      - r6i.8xlarge
-      - r6i.large
-      - r6i.metal
-      - r6i.xlarge
-      - r6id.12xlarge
-      - r6id.16xlarge
-      - r6id.24xlarge
-      - r6id.2xlarge
-      - r6id.32xlarge
-      - r6id.4xlarge
-      - r6id.8xlarge
-      - r6id.large
-      - r6id.metal
-      - r6id.xlarge
-      - u-12tb1.112xlarge
-      - u-3tb1.56xlarge
-      - u-6tb1.112xlarge
-      - u-6tb1.56xlarge
-      - u-9tb1.112xlarge
-      - vt1.24xlarge
-      - vt1.3xlarge
-      - vt1.6xlarge
-      - x2idn.16xlarge
-      - x2idn.24xlarge
-      - x2idn.32xlarge
-      - x2idn.metal
-      - x2iedn.16xlarge
-      - x2iedn.24xlarge
-      - x2iedn.2xlarge
-      - x2iedn.32xlarge
-      - x2iedn.4xlarge
-      - x2iedn.8xlarge
-      - x2iedn.metal
-      - x2iedn.xlarge
-      - x2iezn.12xlarge
-      - x2iezn.2xlarge
-      - x2iezn.4xlarge
-      - x2iezn.6xlarge
-      - x2iezn.8xlarge
-      - x2iezn.metal
+      # updated 2023-06-23
```

Comment:
can you please list the full names? It should be returned by the command

```shell
aws ec2 describe-instance-types \
--filters Name=network-info.encryption-in-transit-supported,Values=true \
--query "InstanceTypes[*].[InstanceType]" --output text | sort
```

Reply:
Done.